src/store/metadata/stub/content/editions/additions-to-the-housing-stock_2023-05.json

```diff
@@ -0,0 +1,162 @@
+{
+  "@context": "https://staging.idpd.uk/ns#",
+  "@id": "https://staging.idpd.uk/datasets/additions-to-the-housing-stock/editions",
+  "@type": "hydra:Collection",
+  "title": "Net additions to the housing stock",
+  "editions": [
+    {
+      "@id": "https://staging.idpd.uk/datasets/additions-to-the-housing-stock/editions/2023-05",
+      "@type": "dcat:Dataset",
+      "in_series": "https://staging.idpd.uk/datasets/additions-to-the-housing-stock",
+      "identifier": "2023-05",
+      "title": "Net additions to the housing stock for 2022 to 2023",
+      "summary": "Net additions to the dwelling stock (including new builds, conversions, changes of use, other gains/losses and offset by demolitions) per 1,000 stock, England, financial year 2022 - 2023.",
+      "description": "This dataset shows the net additions to the dwelling stock (including new builds, conversions, changes of use, other gains/losses and offset by demolitions) per 1,000 stock, in England, for financial year 2022 - 2023.\n\n### Coverage \n\nEngland\n\n#### Geography definition\n\nLocal Authority Districts and Unitary Authorities,\nCounties,\nMetropolitan Counties,\nNation (England) \n\n### Source\n\n - England: [Department for Levelling Up, Housing and Communities]",
```

Comment:
Please remove \n\n### Coverage \n\nEngland\n\n#### Geography definition\n\nLocal Authority Districts and Unitary Authorities,\nCounties,\nMetropolitan Counties,\nNation (England) \n\n### Source\n\n - England: [Department for Levelling Up, Housing and Communities]

This will come under Coverage.